src/titiler/application/pyproject.toml

```diff
@@ -50,6 +50,10 @@ test = [
 server = [
     "uvicorn[standard]>=0.12.0",
 ]
+telemetry = [
```

Comment:
so `titiler.application` is a **demo** application so IMO it's fine if it comes with `battery included` meaning that I'll be fine to have `telemetry` packages shipped with it by default. 

if users do not want the telemetry packages they can easily build their own application with `titiler.core` without needing `titiler.application`